crates/polars-sql/src/sql_expr.rs

```diff
@@ -122,6 +129,7 @@ pub enum SubqueryRestriction {
 /// Recursively walks a SQL Expr to create a polars Expr
 pub(crate) struct SQLExprVisitor<'a> {
     ctx: &'a mut SQLContext,
+    active_schema: Option<SchemaRef>,
```

Comment:
I think we can borrow here. `Option<&'a Schema>`.

Reply:
Done (I think :))